keras/layers/preprocessing/index_lookup.py

```diff
@@ -88,11 +88,11 @@ class IndexLookup(Layer):
 
     def __init__(
         self,
-        max_tokens,
-        num_oov_indices,
-        mask_token,
-        oov_token,
-        vocabulary_dtype,
+        mask_token=None,
+        oov_token="[OOV]",
+        vocabulary_dtype="string",
+        num_oov_indices=1,
+        max_tokens=None,
```

Comment:
The order of the arguments here should follow the order listed in the docstring.